Mathlib/Combinatorics/SimpleGraph/Copy.lean

```diff
@@ -1,43 +1,64 @@
 /-
-Copyright (c) 2025 Mitchell Horner. All rights reserved.
+Copyright (c) 2023 Yaël Dillies, Mitchell Horner. All rights reserved.
```

Comment:
@eric-wieser do you remember the correct way to changing copyright like this?

Reply:
This is probably fine